src/plugins/event-plugins/JsErrorPlugin.ts

```diff
@@ -90,4 +96,15 @@ export class JsErrorPlugin implements Plugin {
             this.promiseRejectEventHandler
         );
     }
+
+    private isErrorInIgnoreList(message: string): boolean {
+        for (const pattern of this.config.errorsToExclude) {
```

Comment:
I have to mention that you've defined `errorsToExclude?: RegExp[];` as an option property, which means you should make an early exit if errorsToExclude is not provided. Although `defaultConfig` provided one array literal but that still not making sense in semantics.

Reply:
This is a good point! Hopefully this is no longer an issue :)